Test/Install-WhiskeyNode.Tests.ps1

```diff
@@ -81,7 +85,15 @@ function ThenNodeInstalled
         $platformID = 'darwin'
         $extension = 'tar.gz'
     }
-    Join-Path -Path $testRoot -ChildPath ('.node\node-{0}-{1}-x64.{2}' -f $NodeVersion,$platformID,$extension) | Should -Exist
+
+    if($NodeZipFileCheck)

```

Comment:
Nit: our coding standard is to put a space after the opening `(` and before the closing `)`), e.g. `if( $NodeZipFileCheck )`. Please update all expressions to use this formatting.